.storybook/stories/card.stories.js

```diff
@@ -0,0 +1,125 @@
+import FlexGridCol from "../../components/flexGrid/FlexGridCol";
+import FlexGridRow from "../../components/flexGrid/FlexGridRow";
+import CardContainer from "../../components/cards/CardContainer";
+import CardBody from "../../components/cards/CardBody";
+import CardButton from "../../components/cards/CardButton";
+import CardHero from "../../components/cards/CardHero";
+import React from "react";
+
+const imageLink = "./img/top-bowie-state-combined-image.svg";
+const dummyText =
+  "Pudding jelly beans tiramisu bear claw apple pie liquorice. Jelly beans macaroon wafer topping gummies. Shortbread tiramisu lollipop topping apple pie tart pie chocolate. Tart croissant cake danish wafer icing sugar plum.";
+
+export default {
+  title: "Card",
+  component: CardContainer,
+  subcomponent: { CardBody },
+  argTypes: {
+    variant: {
+      options: ["elevated", "outline", ""],
+      control: { type: "radio" },
+    },
+  },
+};
+const Template = (args) => (
+  <FlexGridRow hasGutter gutterSize={32}>
+    <FlexGridCol>
+      <CardContainer {...args}></CardContainer>
+    </FlexGridCol>
+  </FlexGridRow>
```

Comment:
you can set the width of the cards here so they're not so wide.  you can set the number columns as a prop on the FlexGridCol component.  we're using a 12 column grid so the following set the desktop width to 4 columns,  on tablet 2 columns and then 1 column on mobile.

<FlexGridCol desktop={4} tablet={2} mobile={1}> ....